exercises/custom_enum_with_reduce.livemd

```diff
@@ -107,6 +110,10 @@ defmodule CustomEnum do
     ["2", "3"]
   """
   def filter(list, callback_function) do
+    Enum.reduce(list, [], fn elem, acc ->
+      if callback_function.(elem), do: [elem | acc], else: acc
+    end)
+    |> CustomEnum.reverse()
```

Comment:
When calling a function from within the module you can omit the Module name

ie. CustomEnum.reverse() -> reverse()

Reply:
That somehow slipped my mind. Thanks.